### ISSUE TITLE
mco: clean up operator.go logging / mco logs

### ISSUE THREAD

**Issue by kikisdeliveryservice** [openshift/machine-config-operator#345]
The new osImageURL logging in operator.go fills the logs with hundreds of:
`I0124 17:35:43.070627       1 operator.go:270] using osimageurl: registry.svc.ci.openshift.org/openshift/origin-v4.0-2019-01-23-155935@sha256:61dc83d62cfb5054c4c5532bd2478742a0711075ef5151572e63f94babeacc1a
I0124 17:35:47.628086       1 operator.go:270] using osimageurl: registry.svc.ci.openshift.org/openshift/origin-v4.0-2019-01-23-155935@sha256:61dc83d62cfb5054c4c5532bd2478742a0711075ef5151572e63f94babeacc1a
I0124 17:36:00.896894       1 operator.go:270] using osimageurl: registry.svc.ci.openshift.org/openshift/origin-v4.0-2019-01-23-155935@sha256:61dc83d62cfb5054c4c5532bd2478742a0711075ef5151572e63f94babeacc1a`

making it very difficult to read the logs to troubleshoot.  Thinking to move those to v2 or remove them altogether.

**Comment by kikisdeliveryservice** [comment]
@cgwalters Happy to fix this, but what do you think would be best?

**Comment by ashcrow** [comment]
@kikisdeliveryservice Do we know why it's repeating? Is it happening per worker?

**Comment by cgwalters** [comment]
We can just delete that log message.

**Comment by kikisdeliveryservice** [comment]
@ashcrow For clarity: this is just some logging that was added recently during the osImageURL work but that due to being in sync() it logs...often.  I ran into this when I was looking into an issue reported in #346 it was really hard to see anything in the MCO logs due to these lines printed out 100s of times. The osImageURL is available in a few other places so removing this logging would make sense so that the MCO `oc logs -f` are readable. 

@cgwalters Thanks! I'll remove
